spring-cloud-gcp-pubsub-stream-binder/README.adoc

```diff
@@ -0,0 +1,29 @@
+= Spring Cloud GCP Stream for Pub/Sub
+
+This module contains the Spring Cloud GCP Stream for Pub/Sub logic.
+
+== Spring Cloud GCP Stream Config
+
+
+Maven coordinates, using Spring Cloud GCP BOM:
+
+[source,xml]
+----
+<dependency>
+    <groupId>org.springframework.cloud</groupId>
+    <artifactId>spring-cloud-gcp-pubsub-stream-binder</artifactId>
+</dependency>
+----
+
+Gradle coordinates:
+
+[source]
+----
+dependencies {
+    compile group: 'org.springframework.cloud', name: 'spring-cloud-gcp-pubsub-stream-binder'
+}
+----
+
+=== Setup for Pub/Sub Emulator
+
+To run this binder with a local emulator you just need to provide the property https://cloud.google.com/pubsub/docs/emulator?hl=en#manually_setting_the_variables[`PUBSUB_EMULATOR_HOST`]
```

Comment:
Sorry to nag here, but do you think it can also be useful to have a simple instruction here on how to run the emulator, as well as the order in which the actions should be taken?

Reply:
No, I think this is already documented well at the linked page